src/include/raft_net.h

```diff
@@ -325,7 +325,7 @@ struct raft_net_client_request_handle
     const struct raft_client_rpc_msg     *rncr_request;
     const char                           *rncr_request_or_commit_data;
     const size_t                          rncr_request_or_commit_data_size;
-    struct raft_client_rpc_msg           *rncr_reply;
+    const char                           *rncr_reply;
```

Comment:
Should we keep this and add a new structure member for the `WritePrep()` operation?

Reply:
Also.. you can keep the type for the `rncr_reply` and not have to cast it based on the context, which is arguably much less safe than what I'm suggesting.